cmd/finch/virtual_machine_settings.go

```diff
@@ -0,0 +1,83 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/runfinch/finch/pkg/config"
+	"github.com/runfinch/finch/pkg/flog"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newSettingsVMCommand(
+	logger flog.Logger,
+	lca config.LimaConfigApplier,
+	fs afero.Fs,
+	stdout io.Writer,
+) *cobra.Command {
+	settingsVMCommand := &cobra.Command{
+		Use:   "settings",
+		Short: "Configure the virtual machine instance",
+		RunE:  newSettingsVMAction(logger, lca, fs, stdout).runAdapter,
+	}
+
+	settingsVMCommand.Flags().Int("cpus", 0, "the amount of vCPU to dedicate to the virtual machine (restart the vm when applying this change.)")
```

Comment:
what happens if we pass in a configure thats not supported does it throw an error?

Reply:
Yes, it will throw an error.
```
haytok ~/workspace/finch [main]
> /Users/haytok/workspace/finch/_output/bin/finch vm settings --foo
FATA[0000] unknown flag: --foo
haytok ~/workspace/finch [main]
> /Users/haytok/workspace/finch/_output/bin/finch vm settings --foo=bar
FATA[0000] unknown flag: --foo
```